src/olympia/reviewers/views.py

```diff
@@ -1132,8 +1136,8 @@ def unsubscribe(self, request, **kwargs):
     def disable(self, request, **kwargs):
         addon = get_object_or_404(Addon, pk=kwargs['pk'])
         ActivityLog.create(amo.LOG.CHANGE_STATUS, addon, amo.STATUS_DISABLED)
-        self.log.info('Addon "%s" status changed to: %s' %
-                      (addon.slug, amo.STATUS_DISABLED))
+        self.log.info('Addon "%s" status changed to: %s',
+                      addon.slug, amo.STATUS_DISABLED)
```

Comment:
hmm?  This isn't a `%` substitution?

Reply:
It is, but when logging, it's better not to do the formatting yourself, and let logging do that with the arguments you pass. It allows whatever log processing you have to group similar log statements together instead of considering each one as a separate kind of event. Sentry does that for instance (not that it matters for `info` calls, but it's best to be coherent about this, so I try to fix it whenever I see it)